django_mongodb/base.py

```diff
@@ -74,30 +74,23 @@ class DatabaseWrapper(BaseDatabaseWrapper):
         "iendswith": "LIKE UPPER(%s)",
     }
     mongo_operators = {
-        "exact": lambda val: val,
-        "gt": lambda val: {"$gt": val},
-        "gte": lambda val: {"$gte": val},
-        "lt": lambda val: {"$lt": val},
-        "lte": lambda val: {"$lte": val},
-        "in": lambda val: {"$in": val},
-        "range": lambda val: {"$gte": val[0], "$lte": val[1]},
-        "isnull": lambda val: None if val else {"$ne": None},
-        "iexact": safe_regex("^%s$", re.IGNORECASE),
-        "startswith": safe_regex("^%s"),
-        "istartswith": safe_regex("^%s", re.IGNORECASE),
-        "endswith": safe_regex("%s$"),
-        "iendswith": safe_regex("%s$", re.IGNORECASE),
-        "contains": safe_regex("%s"),
-        "icontains": safe_regex("%s", re.IGNORECASE),
-        "regex": lambda val: re.compile(val),
-        "iregex": lambda val: re.compile(val, re.IGNORECASE),
-    }
-    mongo_aggregations = {
         "exact": lambda a, b: {"$eq": [a, b]},
         "gt": lambda a, b: {"$gt": [a, b]},
         "gte": lambda a, b: {"$gte": [a, b]},
         "lt": lambda a, b: {"$lt": [a, b]},
         "lte": lambda a, b: {"$lte": [a, b]},
+        "in": lambda a, b: {"$in": [a, b]},
+        "isnull": lambda a, b: {"$eq" if b else "$ne": [a, None]},
```

Comment:
I think it would be readability to add parenthesis: `("$eq" if b else "$ne")`

Reply:
Done